packages/mock/src/msw/index.ts

```diff
@@ -103,9 +108,10 @@ export const ${handlerName} = (${isReturnHttpResponse && !isTextPlain ? `overrid
 
   return {
     implementation: {
-      function: isReturnHttpResponse
-        ? `export const ${functionName} = (${isResponseOverridable ? `overrideResponse: any = {}` : ''}): ${returnType} => (${value})\n\n`
-        : '',
+      function:
+        value && value !== 'undefined'
```

Comment:
common methods can be used.

```suggestion
        isReturnHttpResponse
```

Reply:
Oh it seems changed by missing conflict-resolving sorry!

Fixed in 3d152bbef1389566d37331370a3fdf2bf6588cb1

PTAL